src/objective-c/tests/Tests.xcodeproj/xcshareddata/xcschemes/PerfTestsPosix.xcscheme

```diff
@@ -0,0 +1,85 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Could you remind me of the reason for having a separate scheme here?

Reply:
The separate scheme is to ensure that CFStream and Posix client tests are run separately. Choosing the TCP client implementation (CFStream or Posix) is determined by an env var which is read when the gRPC is initialized.

Reply:
Ah right. Thanks. FYI I don't see the env var being set in this scheme. Double check it

Reply:
The env var is set in the tests: PerfTests/PerfTestsNoCFStreamSSL.m, PerfTests/PerfTestsCFStreamSSL.m